ras_party/models/models.py

```diff
@@ -139,7 +156,12 @@ def _get_attributes_for_collection_exercise(self, collection_exercise_id=None):
                 if attributes.collection_exercise == collection_exercise_id:
                     return attributes
 
-        return self.attributes[-1]
+        try:
+            return next((attributes for attributes in self.attributes if attributes.collection_exercise))
+        except StopIteration:
+            logger.error("No active attributes for business", party_id=self.party_uuid)
```

Comment:
Can we add the status code to this log line please as this will help in the splunk logs.